src/include/enclave/enclave_mem.h

```diff
@@ -29,7 +29,7 @@ void* enclave_mremap(
 
 int enclave_mmap_flags_supported(int flags, int fd);
 
-int syscall_SYS_sysinfo(struct sysinfo* info);
+long syscall_SYS_sysinfo(struct sysinfo* info);
```

Comment:
Please can you pull the overridden syscalls' prototypes into a separate header and move the implementation into the `src/lkl/` directory?  These are LKL-specific, not generic support code.

Reply:
The overriding function uses static variables from `enclave_mem.c`. What's the recommended way to share variables between `src/lkl` and `src/enclave`?